### DISCUSSION TITLE
Image saving

### DISCUSSION THREAD

**Discussion by ghoffart** [hyperspacedev/hyperspace#238] [Ideas]
When looking at a Toot with an embedded image, why can’t I drag&drop the image to the desktop? Or right-click -> Save? Share? That’d be super handy.

As a workaround right now I have to open the Toot in the web view in a web browser, and then grab the image.

**Comment by alicerunsonfedora** [comment]
There currently is no way to save an image in Hyperspace Desktop since this feature was never implemented; however, we could implement it in the future once our team has expanded a bit.
